### PR TITLE
EZP-28922: Design consistency for modals across the whole application

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-28922](https://jira.ez.no/browse/EZP-28922)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Design consistency for modals across the whole application:
- Updated styling for modals (Translations, Send to Trash and Send to Trash from Trash manager), background-colors.
- In Translations modal aligned colors for language selection for translations to the existing one in content-edit-language widget.

screen shot 2018-03-08 at 1 28 06 pm
screen shot 2018-03-08 at 1 40 34 pm
screen shot 2018-03-08 at 1 41 56 pm

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/_add-translation.scss [hunk 1]
```diff
@@ -7,15 +7,28 @@
...
         }
     }
 
     .modal-header {
-        background: $ez-ground-base;
+        background: $ez-color-base-pale;
+        border-bottom: none;
+
+        .close {
+            margin-top: -.55rem;
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_add-translation.scss]: If the root font size is `16px` then `.55rem` is `8.8px`. Maybe it should be `.5625rem` (`9px`) to make it rounded to the nearest number.

File: src/bundle/Resources/public/scss/_modals.scss [hunk 2]
```diff
@@ -1,9 +1,13 @@
 .ez-modal--send-to-trash {
     .modal-dialog {
-        background-color: $ez-ground-base-medium;
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_modals.scss]: I would keep it here.

File: src/bundle/Resources/views/content/tab/translations/modal_add_translation.html.twig [hunk 3]
```diff
@@ -6,7 +6,7 @@
         {{ form_start(form, {'action': path('ezplatform.translation.add')}) }}
         <div class="modal-content">
             <div class="modal-header">
-                <h5 class="modal-title">{{ 'tab.translations.add.title'|trans|desc('Create new translation') }}</h5>
+                <h2 class="modal-title">{{ 'tab.translations.add.title'|trans|desc('Create new translation') }}</h2>
```
Comment (sunpietro) [inline src/bundle/Resources/views/content/tab/translations/modal_add_translation.html.twig]: We need to think about semantics here. If you set it to `h2` then it semantically means it is a second important title in the HTML markup. If you would like to resize the title. Then maybe you should create a CSS selector that provides a desired font size definition.

**Comment by inakijv** [comment]
Thanks for your review @sunpietro :-)
Updated PR based on it.

**Pull request merged**